truffle-seq/OwnerManagedLite_test_add_owner.md

```diff
@@ -0,0 +1,1329 @@
+<#FEFECE>| OwnerManagedLite_01 | OwnerManagedLite | 0x3Ac8934DbD2dd808d8E5A25151D9f4874279BEae |
+endlegend
+
+@enduml
+```
+
+##### d3, tx: 0x1c5dc1253a074ba528f9a7e2a7730d8d4a0e8fd553860686b1d90d2edc4d29ab
+
+[SVG :telescope:](https://www.planttext.com/api/plantuml/svg/xLLTRzCm57tthx36owOny5SSUnhLsYPbaq2Ts2HU9Y4dTjjegLib3hHR-U_SfgNQH46Yuf4eyhNllJwspurF4BgiWwb2CoqHCasO-MQQkGhL3uMVcyfCSUd6pjkUoH_6rQpnjZyhPnL-DKW7QJz5wA8Eoz9r46wVYJFMLVXfEyJuQbOLZpCVJDajYx6VEXy0SHHs2GC8RY3dJJKltOl0RV7eC1Kxtusnjl1ZJCdMzRsr5s-tcq3RzQzzNOmdeSRP4jzLpMW4k2WKeNJuRk7nbQadvvWiQ1xPdBA86nABp4HCZRI97JDnp8bLLXZYrCX64LUII2KpQZMnpDbSMAPDXe2oFC3Mqs4NcneVWJr2G5ieyc9kuAZ3RzvL7uqtOsS_5C5z8RHDtEC_GgYTZyyw--FuzFGS0m6tmGJN9XpV-tjqiIt5wwtjhDxXi9mx3DQRQMk_chA1pZfzjOQeN5tZ5PwjGRgx8LbaKh4qveDHpXNDTSG5rKwnGIwpGMHJ9NaUQzer6wWJ_8J96uBVVqwleHBxD_t7qy32BlGdne_TqBVdEZxvMvopitFyoOMcylWmFfe22hN2yCOHYoKLH6h9DQMn417X95BW4ZgA9QUQ4Yc5r8n55494g9X4EcQ4IYRXeo6IAo4uYsEvvUWGMgRcmGshsv_K71y4ahTp3oJrVt5-Uvn_F6TpvTSvQAEqwEQPVbWNJ57MptXBI488MVC6WWfI0fq-A6K5GhHZS7UbT_CwgrQZe5q4LtbJbijMcopkvZceUcYlVouv1ndanD14o9xiwK6cL9BGf4TY9gKJN0fbr4lWVHhtgxTTW7TpfRb8ieHPguYoAeswKAs89de4vH8irhtKG3829hPye4jeuV_q0m00)
+
+
+```plantuml
+
+
+@startuml
+
+autonumber
+skinparam legendBackgroundColor #FEFECE
+
+<style>
+      header {
+        HorizontalAlignment left
+        FontColor purple
+        FontSize 14
+        Padding 10
+      }
+    </style>
+
+header Insights by Truffle
+
+title Txn Hash: 0x1c5dc1253a074ba528f9a7e2a7730d8d4a0e8fd553860686b1d90d2edc4d29ab
+
+
+actor EOA as "EOA"
+participant OwnerManagedLite_01 as "OwnerManagedLite_01"
+
+"EOA" -> "OwnerManagedLite_01" ++: addStateOwner(\n\
+<#FEFECE,#FEFECE>|= type |= name |= value |\n\
+| address | ownerAddress | 0xb682E73Ffc381c953419e82Fc6bF5dE863c791Aa |\n\
+) { 0.0 GWEI }
+"OwnerManagedLite_01" -> "OwnerManagedLite_01" ++: resetChangeOnState()
+"OwnerManagedLite_01" -> "OwnerManagedLite_01" --: Return (\n\
+<#FEFECE,#FEFECE>|= type |= name |= value |\n\
+| uint |  | 75276140696391174450305814049576319106646922510300487059720162673006384432776 |\n\
+)
+"OwnerManagedLite_01" -> "OwnerManagedLite_01" ++: makeOrSetChange()
+"OwnerManagedLite_01" -> "OwnerManagedLite_01" ++: changeState()
+"OwnerManagedLite_01" -> "OwnerManagedLite_01" ++: resetChangeOnState()
+"OwnerManagedLite_01" -> "OwnerManagedLite_01" --: Return (\n\
+<#FEFECE,#FEFECE>|= type |= name |= value |\n\
+| uint |  | 75276140696391174450305814049576319106646922510300487059720162673006384432776 |\n\
+)
+"OwnerManagedLite_01" -> "OwnerManagedLite_01" --: 
+"OwnerManagedLite_01" -> "OwnerManagedLite_01" --: 
+"OwnerManagedLite_01" -> "EOA" --: 
+
+legend
+Participant details
+<#FEFECE,#D0D000>|= Alias |= Contract name |= Address |
+<#FEFECE>| EOA | Externally Owned Account | 0xc633ba530a1D46B6B9Fb88DD1DB07266e43648a8 |
+<#FEFECE>| OwnerManagedLite_01 | OwnerManagedLite | 0x3Ac8934DbD2dd808d8E5A25151D9f4874279BEae |
+endlegend
+
+@enduml
+```
+
+
+
+## Adds the owner and emits event
```

Comment:
This works as expected 👍 - same comment as the last test and previous one about seeing events